components/user_settings/general/user_settings_general.jsx

```diff
@@ -125,6 +120,78 @@ class UserSettingsGeneralTab extends React.Component {
         this.state = this.setupInitialState(props);
     }
 
+    componentWillMount() {
```

Comment:
`componentWillMount` is actually being deprecated, mind moving this to the constructor to make upgrading React easier in the future?

Reply:
Good to know, updated!